.editorconfig

```diff
@@ -0,0 +1,10 @@
+﻿[*.{cs,vb}]
```

Comment:
Consider just copying the same editorconfig/etc. files from eg. ComputeSharp?

Reply:
So much is just copied from ComputeSharp lol. Should make a repo template from it lol